libs/ui/project.json

```diff
@@ -1,15 +1,16 @@
 {
-	"name": "ui",
-	"$schema": "../../node_modules/nx/schemas/project-schema.json",
-	"sourceRoot": "libs/ui/src",
-	"projectType": "library",
-	"tags": [],
-	"// targets": "to see all targets run: nx show project ui --web",
-	"targets": {
-		"storybook": {
-			"options": {
-				"port": 6006
-			}
-		}
-	}
+  "name": "ui",
```

Comment:
Why is there changes marked here if nothing really changed? Perhaps spacing switching from tabs to spaces. If so would recommend making a separate PR where all these places are fixed. Since it will make it easier to review :)

Reply:
@jsonDoge this PR is done! Applied formatting to the whole project